tests/src/tools/ReadyToRun.SuperIlc/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+namespace ReadyToRun.SuperIlc
+{
+    class Program
+    {
+        static void ShowUsage()
+        {
+            Console.WriteLine("dotnet SuperIlc --in <FolderToCompile> --out <OutputFolder> --ref <Referenced assembly folder> [--cpaot] [--crossgen]");
```

Comment:
Non-blocking - would there be a way to make it so that "inputPath" can arbitrarily refer to a folder or a single file to e.g. let me selectively recompile a single framework assembly? Even better, if we could optionally specify a recursive subtree scan, that would be a strict improvement in comparison with current Pri# 1 CoreCLR testing where I need to hack the scripts manually when I want to build just a subset (e.g. the 1.5K typeloader tests, not the entire 10K test file).

Reply:
Yes, I want single file too. I'll add that if I get some spare time today (separate PR)